tools/ddprobe.cpp

```diff
@@ -149,5 +149,6 @@ int main(int argc, char *argv[]) {
     }
   }
 
-  return 0;
+  // In order to prevent potential false positives, we will return an error here.
+  return -1;
```

Comment:
I actually confirmed this false positive in my own testing, passing in an invalid display name will return back 0... causing the GPU preference to pick the wrong adapter. 

I had to change this to -1 in order to have it properly use the correct adapter in my setup.